ui/perfherder/helpers.js

```diff
@@ -746,7 +746,9 @@ export const createGraphData = (seriesData, alertSummaries, colors, symbols) =>
       projectId: series.repository_id,
       id: `${series.repository_name} ${series.name}`,
       data: series.data.map((dataPoint) => ({
-        x: new Date(dataPoint.push_timestamp),
+        // Backend implicitly provides all dates as UTC.
+        // Let's make this explicit, so frontend doesn't get confused.
+        x: new Date(`${dataPoint.push_timestamp}Z`),
```

Comment:
What is the `Z` for?

Reply:
The literal "Z" is actually part of the ISO 8601 datetime standard for UTC times. When "Z" (Zulu) is tacked on the end of a time, it indicates that that time is UTC, so really the literal Z is part of the time.

Reply:
thanks!